src/services/sdk.ts

```diff
@@ -106,8 +117,18 @@ export const startPostSubscription = (
   dispatch(clearFeedItems());
 
   // subscribe to all announcements
+  let blockNumber: number;
+  let blockIndex = 0;
   core.contracts.subscription.subscribeToBatchPublications(
-    handleBatchAnnouncement(dispatch),
+    (announcement: BatchPublicationLogData) => {
+      if (announcement.blockNumber !== blockNumber) {
+        blockNumber = announcement.blockNumber;
+        blockIndex = 0;
```

Comment:
what is block index?

Reply:
blockIndex is order this batch announcement is received within the block. Announcements found in batches that are published earlier in the block are considered to have come earlier according to the spec.